### PR TITLE
Fixed United Kingom's 2022 holidays

### PR DESCRIPTION
**Pull request by mountinash789**
Queens Platinum Jubilee has been added to 3rd of June and the Spring bank holiday has been moved to the 2nd of June

- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Changelog amended with a mention describing your changes. **Note** *Please do NOT change the version number here. It's the project maintainers' duty.*

### CODE REVIEW

**Comment by ewjoachim** [comment]
Source: https://www.royal.uk/platinum-jubilee-central-weekend

**Comment by brunobord** [comment]
Could you please rebase from upstream in order to fix Changelog conflicts?

**Comment by brunobord** [comment]
As a matter of fact, I think you could wait a bit longer. The next PR to be added will be the #675, so I'll tell you when upstream will be ready to be used as base.

**Comment by brunobord** [comment]
ok, so now you have my green light to rebase from upstream. If you need any help doing these, please do not hesitate to ask ;o)

**Comment by mountinash789** [comment]
@brunobord Is that ok?

**Review by Natim — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Nothing here yet.
 - Remove unused `JalaliMixin`
 - Replace `pkg_resources` with `importlib_metadata` to fetch the version number in `__init__.py` (#657)
 - Added new badges (pypi, conda, license) and installation instructions (pip, conda) to readme file @sugatoray (#673).
+- Replace `pyCalverter` with `convertdate` (#536)
+- Remove unused `JalaliMixin`
+- Fixed United Kingdom's 2022 holidays ; Spring Bank Holiday has been moved to the 3rd of June and Queen's Platinum Jubilee added to 2nd of June.
```
Comment (brunobord) [inline Changelog.md:24]: Sorry, but this changelog entry should be in the "master (unreleased)" section at the top of the file.

File: Changelog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ Nothing here yet.
 - Remove unused `JalaliMixin`
 - Replace `pkg_resources` with `importlib_metadata` to fetch the version number in `__init__.py` (#657)
 - Added new badges (pypi, conda, license) and installation instructions (pip, conda) to readme file @sugatoray (#673).
+- Replace `pyCalverter` with `convertdate` (#536)
```
Comment (brunobord) [inline Changelog.md:22]: This line and the line below are duplicated from the lines 18-19 and should be removed from your patch.

**Review by brunobord — CHANGES_REQUESTED** [review]
Something may have gone wrong when you updated from upstream.
Should be an easy fix to commit and move the log entries around.

**Comment by brunobord** [comment]
ouch! a flake8 error :( https://github.com/workalendar/workalendar/pull/677/checks?check_run_id=3704515113

**Comment by brunobord** [comment]
Your fix is finally part of the latest (16.1.0) release:

* https://pypi.org/project/workalendar/16.1.0/
* https://github.com/workalendar/workalendar/releases/tag/16.1.0

Happy upgrading!

**Pull request merged**